src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+body {
+  margin-top: 0;
+  font-weight: 300;
+}
+
+[data-qa*="big"] {
+  margin: 20px 0;
+}
+
+[data-qa*="big"] input {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  padding: 25px 0 26px 62px;
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+}
+
+[data-qa="small"] input {

```

Comment:
you duplicate a lot of styles, combine them into common selector

Reply:
There are still duplicate rule (as url("./images/Search.svg") )
Create common selector like `.input` where you will have common styles. And add modifiers as `--big` or `--small` to make changes